src/shared/conf-parser.c

```diff
@@ -327,6 +327,9 @@ int config_parse(const char *unit,
                         return r;
                 }
 
+                if (strchr(COMMENTS, *buf))
+                        continue;
```

Comment:
i figure this means the matching check in parse_line() could be dropped then?

Reply:
hmm, is this even right? if i do the following

```
foo \
# bar
```
Then this should be considered equivalent to this I think:

```
foo # bar
```

but with your change it would be considered equivalent to 

```
foo
```

Reply:
I thought only `foo` in this case is natural for me... But if you do not like that or you think we should not do that, then I will update this as `foo # bar` for this case.

Reply:
The decision was made to follow bash some time ago. I was against...